stable_baselines3/common/utils.py

```diff
@@ -88,7 +88,7 @@ def get_schedule_fn(value_schedule: Union[Callable, float]) -> Callable:
     return value_schedule
 
 
-def get_linear_fn(start: float, end: float, end_fraction: float) -> Callable:
+def get_linear_fn(start: float, end: float, end_fraction: float) -> Callable[[float], float]:
```

Comment:
same, maybe define a `Schedule` type alias

Reply:
I will go with `LearningRateSchedule` since we might have other kinds of schedules in the future.

Reply:
in fact, for PPO, there is already the clipping ratio which could be a schedule (currently not implemented)
and in DQN, you have the exploration schedule for the epsilon-greedy policy